TestLib/EarlGreyImpl/EarlGreyImpl+XCUIApplication.m

```diff
@@ -54,7 +54,7 @@ - (BOOL)backgroundApplication {
   return [condition waitWithTimeout:10.0 pollInterval:kPollInterval];
 #else
   NSString *errorDescription =
-      @"Cannot perform backgrounding because it is not supported with the current system version.";
+      @"Cannot perform backgrounding because it is not supported with the current system version.\nUse at least iOS 11.";
```

Comment:
@"Cannot perform backgrounding because it is not supported with the current system version."
@"Use at least iOS 11.";